tests/func/test_checkout.py

```diff
@@ -436,10 +436,10 @@ def test(self):
         self.assertEqual(ret, 0)
 
         self.assertTrue(System.is_symlink(self.FOO))
-        old_foo_link = os.readlink(self.FOO)
+        old_foo_link = os.path.realpath(self.FOO)
```

Comment:
Using realpath due to change in how readlink works in Python3.8: https://docs.python.org/3.8/library/os.html#os.readlink)